examples/simple/package.json

```diff
@@ -1,49 +1,26 @@
 {
-    "name": "simple",
-    "private": true,
-    "version": "3.0.0",
-    "description": "",
-    "main": "index.html",
-    "scripts": {
-        "dev": "./node_modules/.bin/webpack-dev-server --progress --color --hot --watch --mode development",
-        "start": "USE_ALIAS=true ./node_modules/.bin/webpack-dev-server --progress --color --hot --watch --mode development",
-        "serve": "./node_modules/.bin/serve --listen 8080 ./dist",
-        "build": "./node_modules/.bin/webpack-cli --color --mode development --hide-modules true"
-    },
-    "author": "",
-    "license": "MIT",
-    "devDependencies": {
-        "@babel/cli": "^7.10.5",
-        "@babel/core": "^7.11.4",
-        "@babel/plugin-proposal-class-properties": "^7.10.4",
-        "@babel/plugin-proposal-object-rest-spread": "^7.11.0",
-        "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-        "@babel/preset-env": "^7.11.0",
-        "@babel/preset-react": "^7.10.4",
-        "@babel/preset-typescript": "^7.10.4",
-        "babel-loader": "^8.0.4",
-        "html-loader": "~1.3.0",
-        "html-webpack-plugin": "~4.3.0",
-        "ignore-not-found-export-plugin": "^1.0.1",
-        "serve": "^10.1.2",
-        "style-loader": "~1.2.1",
-        "wait-on": "^3.2.0",
-        "webpack": "~4.5.0",
-        "webpack-bundle-analyzer": "^3.3.2",
-        "webpack-cli": "~2.0.13",
-        "webpack-dev-server": "~3.1.11"
-    },
-    "dependencies": {
-        "@babel/polyfill": "^7.10.4",
-        "@material-ui/core": "^4.11.2",
-        "@material-ui/icons": "^4.11.2",
-        "ra-data-fakerest": "^3.9.0",
-        "ra-i18n-polyglot": "^3.9.0",
-        "ra-input-rich-text": "^3.9.0",
-        "ra-language-english": "^3.9.0",
-        "ra-language-french": "^3.9.0",
-        "react": "^17.0.0",
-        "react-admin": "^3.9.0",
-        "react-dom": "^17.0.0"
-    }
+  "name": "simple-vite",
+  "version": "0.0.0",
+  "scripts": {
+    "start": "vite",
```

Comment:
the dev script is used by codesandbox. Can you check that it can work with the vite version of the simple example?

Reply:
Ok codesandox will prefer `dev` over `start` but will use `start` if `dev` is not defined. However as I was unable to make Vite work on codesandbox, I added a webpack setup and a `dev` command